client/src/components/middle-bar.jsx

```diff
@@ -91,7 +91,20 @@ class MiddleBar extends React.Component {
       })
     }
 
+    // for review button
+    // i think state is updating but button is not rerendering on state change. maybe because its defined differently?
+    ReviewClickHandler() {
+      // if cost > balance set state error = true
+      // else ?
+      console.log(this.props.price * this.state.shares, this.props.balance)
```

Comment:
You can delete this console.log and the one on line 101 if you aren't  using them for anything other than testing for your own knowledge